README.md

```diff
@@ -1,757 +1,77 @@
-sectors may have different operational constraints (e.g., duration between demand
-curtailment and recovery, directionality of load shift). EFS flexibility data can be
-split by sector as follows:
-```python
-from prereise.gather.demanddata.nrel_efs.get_efs_data import partition_flexibility_by_sector
+## Dependencies
+**PreREISE** relies on:
+* **PowerSimData**, another package developed by Breakthrough Energy Sciences and
+available [here][PowerSimData]. You must have **PowerSimData** cloned in a folder
+adjacent to your clone of **PreREISE** as the installation of packages depends on
+files in **PowerSimData**.
+* Several Python packages all available on [PyPi](https://pypi.org/) whose list can be
+found in the ***requirements.txt*** or ***Pipfile*** files both located at the root of
+this package.
 
-sect_dem = partition_flexibility_by_sector(es, ta, flex, year, sect, fpath, save)
-```
-where `es` is a string describing a single electrification scenario, `ta` is a string
-describing a single technology advancement, flex is a string describing a single
-flexibility scenario, `year` is an integer describing the included year, `sect` is a set
-of the sectors to include, `fpath` is the file path where the flexibility data might be
-located and to where the flexibility demand will be saved (if desired), and `save` is a
-boolean that indicates whether the sectoral flexibility data should be saved. `sect`
-defaults to including all the sectors (i.e., all sectoral flexibility is retained).
-`fpath` defaults to the current directory. `save` defaults to `False` (i.e., each
-sectoral flexibility DataFrame is not saved). `partition_flexibility_by_sector` returns
-`sect_flex`, which is a dictionary of DataFrames, where each DataFrame contains the
-flexibility data for one sector.
 
-`partition_demand_by_sector` and `partition_flexibility_by_sector` check the file path
-provided by `fpath` to determine if the data is already downloaded and extracted. If the
-data is not present as a ***.csv*** file, then `partition_demand_by_sector` and
-`partition_flexibility_by_sector` use `download_demand_data` and
-`download_flexibility_data`, respectively, to obtain the specified ***.csv*** file. If
-the automated approach is not able to extract the ***.csv*** file,
-`partition_demand_by_sector` and `partition_flexibility_by_sector` will exit with an
-error, and the user will need to manually use their extraction method of choice (as was
-discussed in the prior subsection). If manual extraction is required, the user can
-simply run `partition_demand_by_sector` and `partition_flexibility_by_sector` again,
-making sure that `fpath` points to the appropriate location where the ***.csv*** file is
-saved.
+## Installation
+Modules located in the **PowerSimData** package are imported. In particular, profiles
+generation relies heavily on `Grid` objects. Follow the instructions in the README of
+the [PowerSimData] package to install our software ecosystem.
```

Comment:
Should we also give a clue as to how to install the other dependencies?

Reply:
Yeah, it is confusing. Instead of:

---

## Dependencies
**PreREISE** relies on:
* **PowerSimData**, another package developed by Breakthrough Energy Sciences and
available here. You must have **PowerSimData** cloned in a folder
adjacent to your clone of **PreREISE** as the installation of packages depends on
files in **PowerSimData**.
* Several Python packages all available on [PyPi](https://pypi.org/) whose list can be
found in the ***requirements.txt*** or ***Pipfile*** files both located at the root of
this package.


## Installation
Modules located in the **PowerSimData** package are imported. In particular, profiles
generation relies heavily on `Grid` objects. Follow the instructions in the README of
the [PowerSimData] package to install our software ecosystem.

---

We could have:

---

## Dependencies
**PreREISE** relies on:
* **PowerSimData**, another package developed by Breakthrough Energy Sciences and
available here.
* Several Python packages all available on [PyPi](https://pypi.org/) whose list can be
found in the ***requirements.txt*** or ***Pipfile*** files both located at the root of
this package.


## Installation
Clone **PowerSimData** in a folder adjacent to your clone of **PreREISE** as the installation of packages depends on
files in **PowerSimData**. Then, use the **requirements.txt** file or **Pipfile** to install third party dependencies.

---

What do you think?

Reply:
I like it.

Reply:
@danielolsen, what do you think?

Reply:
I like it too. Maybe making it even explicit:
"Then, use the requirements.txt file (via `pip`) or Pipfile (via `pipenv`) to install third party dependencies."

Reply:
Done